### PR TITLE
Add io.gitlab.Turtlico

### PR DESCRIPTION
**Pull request by saytamkenorh**
This is a resubmission of [tk.turtlico.Turtlico](https://github.com/flathub/tk.turtlico.Turtlico). The app ID has changed due to transfer of the project's website to a new domain.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build io.gitlab.Turtlico

File: io.gitlab.Turtlico.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
```
Comment (bilelmoussaoui) [inline io.gitlab.Turtlico.json:31]: It's already on the SDK

**Comment by saytamkenorh** [comment]
bot, build io.gitlab.Turtlico

**Comment by barthalion** [comment]
/merge

Please remember to add end-of-life and end-of-life-rebase to https://github.com/flathub/tk.turtlico.Turtlico. Poke me if it fails to build, I'll handle it manually in that case.

https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

**Comment by saytamkenorh** [comment]
@barthalion I added end-of-life and end-of-life-rebase but the build failed due to invalid appstream (it contains a link to a screenshot on the old website). Would you please handle it manually then?

**Comment by barthalion** [comment]
Sure thing.

**Pull request closed without merging**